docs/admin/migration.md

```diff
@@ -1,17 +1,81 @@
 
-!!! note
-    The migration job runs in kserve namespace. please make sure kserve running in the cluster.
 
+### Migrating from standalone KFServing
 
-### Migration
+1. Install KServe v0.7 using the [install YAML](https://github.com/kserve/kserve/blob/master/install/v0.7.0/kserve.yaml)
+    - This will not affect existing services yet.
 
-Please use this [YAML file](./kserve_migration_job.yaml) to migrate existing inference services.
+    ```bash
+    kubectl apply -f https://github.com/kserve/kserve/blob/master/install/v0.7.0/kserve.yaml
+    ```
 
-=== "kubectl"
-```bash
-kubectl apply -f kserve_migration_job.yaml
-```
+2. Run the [KServe Migration YAML](https://github.com/kserve/kserve/blob/master/hack/kserve_migration/kserve_migration_job.yaml)
+    - This will begin the migration. Any errors here may affect your existing services.
+
+    - If you do not want to delete the KFServing resources after migrating, download and edit the env `REMOVE_KFSERVING`
+      in the YAML before applying it 
+
+    - If your KFServing is installed in a namespace other than `kfserving-system`, then download and set the env `KFSERVING_NAMESPACE`
+      in the YAML before applying it
+
+    ```bash
+    kubectl apply -f https://github.com/kserve/kserve/blob/master/hack/kserve_migration/kserve_migration_job.yaml
+    ```
+
+3. Clean up the migration resources
+
+    ```bash
+    kubectl delete ClusterRoleBinding cluster-migration-rolebinding
+    kubectl delete ClusterRole cluster-migration-role
+    kubectl delete ServiceAccount cluster-migration-svcaccount -n kserve 
+    ```
+
+### Migrating from Kubeflow-based KFServing
+
+1. Install Kubeflow-based KServe 0.7 using the [install YAML](https://github.com/kserve/kserve/blob/master/install/v0.7.0/kserve_kubeflow.yaml)
+    - This will not affect existing services yet.
+
+    ```bash
+    kubectl apply -f https://github.com/kserve/kserve/blob/master/install/v0.7.0/kserve_kubeflow.yaml
+    ```
+
+2. Run the [KServe Migration YAML](https://github.com/kserve/kserve/blob/master/hack/kserve_migration/kserve_migration_job_kubeflow.yaml) for Kubeflow-based installations
+    - This will begin the migration. Any errors here may affect your existing services.
+
+    - If you do not want to delete the KFServing resources after migrating, download and edit the env `REMOVE_KFSERVING`
+      in the YAML before applying it 
+
+    ```bash
+    kubectl apply -f https://github.com/kserve/kserve/blob/master/hack/kserve_migration/kserve_migration_job_kubeflow.yaml
+    ```
+
+3. Clean up the migration resources
+
+    ```bash
+    kubectl delete ClusterRoleBinding cluster-migration-rolebinding
+    kubectl delete ClusterRole cluster-migration-role
+    kubectl delete ServiceAccount cluster-migration-svcaccount -n kubeflow 
+    ```
+
+4. Update the models web app to use the new InferenceService API group `serving.kserve.io`
+    - Change the deployment image to `kserve/models-web-app:v0.7.0-rc0`
```

Comment:
The model webapp does not support the new api group yet right?

Reply:
The web app uses the new api group already
https://github.com/kserve/models-web-app/blob/master/backend/apps/common/versions.py#L25